pom.xml

```diff
@@ -7,6 +7,20 @@
     <groupId>com.aquaq.training</groupId>
     <artifactId>Java-Refresher-Practicals</artifactId>
     <version>1.0-SNAPSHOT</version>
+    <dependencies>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-core</artifactId>
+            <version>4.8.0</version>
```

Comment:
Mockito isn't currently being used but will be on update. Please upgrade to latest version found on https://mvnrepository.com/